docker/datahub-ingestion/Dockerfile

```diff
@@ -1,10 +1,12 @@
 # Defining environment
 ARG APP_ENV=prod
+ARG DOCKER_VERSION=latest
 
-FROM acryldata/datahub-ingestion-base as base
+FROM acryldata/datahub-ingestion-base:$DOCKER_VERSION as base
 
-FROM openjdk:11 as prod-build
+FROM eclipse-temurin:11 as prod-build
```

Comment:
The openjdk image was abandoned last year. This is the recommended replacement. This is a build container so not part of the final container.

Reply:
This is the only place we use an image like this. In other places we are using alpine based images.